assets/enhancements.css

```
@@ -49,45 +49,607 @@
 /* ---------- Code Runner ---------- */
 .code-run-wrap{display:flex;align-items:center;justify-content:flex-end;gap:8px;padding:6px 12px;background:#1a1813;border-top:1px solid rgba(255,255,255,.05);}
-.code-run-btn{display:inline-flex;align-items:center;gap:6px;background:rgba(184,64,46,.14);color:#e0896a;border:1px solid rgba(184,64,46,.25);border-radius:6px;padding:5px 14px;font-family:"JetBrains Mono",monospace;font-size:12px;font-weight:600;cursor:pointer;transition:all .2s;}
-.code-run-btn:hover{background:rgba(184,64,46,.22);color:#f0c8b8;border-color:rgba(184,64,46,.45);}
+.code-run-btn{display:inline-flex;align-items:center;gap:6px;background:rgba(var(--accent-rgb),.14);color:var(--accent);border:1px solid rgba(var(--accent-rgb),.25);border-radius:6px;padding:5px 14px;font-family:"JetBrains Mono",monospace;font-size:12px;font-weight:600;cursor:pointer;transition:all .2s;}
+.code-run-btn:hover{background:rgba(var(--accent-rgb),.22);color:#ffffff;border-color:rgba(var(--accent-rgb),.45);}
 .code-run-btn.loading{opacity:.55;cursor:wait;}
 .code-playground-link{display:inline-flex;align-items:center;gap:6px;color:#7fb0c9;text-decoration:none;font-family:"JetBrains Mono",monospace;font-size:12px;font-weight:600;padding:5px 14px;border-radius:6px;border:1px solid rgba(128,176,208,.18);transition:all .2s;}
 .code-playground-link:hover{background:rgba(128,176,208,.08);border-color:rgba(128,176,208,.35);}
-.code-output-panel{background:#16130f;border-top:1px solid rgba(255,255,255,.05);padding:10px 14px;font-family:"JetBrains Mono",monospace;font-size:12px;line-height:1.55;color:#a9c98a;max-height:200px;overflow-y:auto;white-space:pre-wrap;word-break:break-word;}
-.code-output-panel.error{color:#e0896a;}
+.code-output-panel{background:#0d0e11;border-top:1px solid rgba(255,255,255,.05);padding:10px 14px;font-family:"JetBrains Mono",monospace;font-size:12px;line-height:1.55;color:#a8e6a0;max-height:200px;overflow-y:auto;white-space:pre-wrap;word-break:break-word;}
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)